src/sth_configuration.js

```diff
@@ -70,6 +70,16 @@
   module.exports.SERVICE_PATH = ENV.SERVICE_PATH || '/';
   module.exports.POOL_SIZE = (ENV.POOL_SIZE && !isNaN(ENV.POOL_SIZE) && parseInt(ENV.POOL_SIZE) > 0) ?
     parseInt(ENV.POOL_SIZE) : 5;
+  try {
```

Comment:
This new parameter must be had into account in the `stf_default.conf` file within PR https://github.com/telefonicaid/IoT-STH/pull/45